test/endpoints/automations.test.js

```diff
@@ -16,94 +30,107 @@ describe('Tests for automation endpoints\n', () => {
           .to.equal('Successfully fetched automations');
         expect(res.body)
           .to.have.property('data')
-          .to.be.an('array');
+          .to.be.an('array')
+          .to.have.length(10);
```

Comment:
below 10 can mean zero which will make the test pass even though it maybe a failure